docker-for-windows/edge-release-notes.md

```diff
@@ -12,6 +12,57 @@ For information about Stable releases, see the [Stable release
+- Docker Desktop defaults to WSL 2 on install on compatible OS versions.
+- Docker Desktop detects missing Linux kernel and adds a pointer to the Microsoft documentation to download the kernel.
+
+**File sharing**
+- Kubernetes: Persistent volumes created by claims are now stored in the virtual machine. Fixes [for-win/issues/5665](https://github.com/docker/for-win/issues/5665).
+- Docker Desktop ensures that host paths accessed by containers are within the shared folders list.
+- Fixed a bug where opening a read-only file would fail with an `Operation not permitted error`. Fixes [docker/for-win#6016](https://github.com/docker/for-win/issues/6016) and [docker/for-win#6017](https://github.com/docker/for-win/issues/6017).
+- Fixed path handling for `docker volume create -o type=none -o o=bind -o device=C:\Some\Windows\path`.
+- Fixed a bug which prevented open files from being deleted. Fixes [docker/for-win#5565](https://github.com/docker/for-win/issues/5565).
+- Docker Desktop now avoids locking files on the host which are open in containers.
+- Docker Desktop generates `fsnotify.WRITE` events in Linux containers when files are changed on the host, [docker/for-win#5530](https://github.com/docker/for-win/issues/5530#issuecomment-585572414)
+- Docker Desktop now displays hidden files on shared volumes. Fixes [docker/for-win#5808](https://github.com/docker/for-win/issues/5808).
+- Fixed cache invalidation and event injection for shared volumes with long paths.
+- Docker Desktop now handles case-insensitivity correctly during file creation.
+- Docker Desktop represents valid directory junctions as directories (rather than symlinks) and handles cache invalidation and event injection properly. Fixes [docker/for-win#5582](https://github.com/docker/for-win/issues/5582).
+- Fixed a race condition in `readlink` on shared volumes using "mfsymlinks". Fixes [docker/for-win#5793](https://github.com/docker/for-win/issues/5793)
+- Fixed a bug where files did not change in shared volumes when using `volumes_from` in docker-compose.yml. Fixes [docker/for-win#5530](https://github.com/docker/for-win/issues/5530).
+
+**Other fixes**
+- Added an option to delete data from the **Troubleshoot** screen.
+- Docker Desktop now reads the Hyper-V VM disk max size on startup and uses it as the value to display in the Settings.
+- Fixed the Hyper-V used disk image size being reported incorrectly.
+- Fixed the Hyper-V VM disk size increase. Fixes [docker/for-win#5881](https://github.com/docker/for-win/issues/5881).
+- Fixed container time drift when system hibernates. Fixes [docker/for-win#4526](https://github.com/docker/for-win/issues/4526).
+- Fixed a bug where the Docker Desktop UI could be started without the engine. Fixes [docker/for-win#5376](https://github.com/docker/for-win/issues/5376).
+- Docker Desktop now uses the least possible privilege for querying the LanManServer service. Fixes [docker/for-win#5150](https://github.com/docker/for-win/issues/5150).
```

Comment:
It’s `LanmanServer`

![image](https://user-images.githubusercontent.com/2386884/78023274-a7ee1080-7356-11ea-8eb2-f7229c66671c.png)

but we could probably should say `Server service`.

Reply:
Thanks. Will update it as `Server service`.